### DISCUSSION TITLE
Release notes / changelog: Curate?

### DISCUSSION THREAD

**Discussion by tony** [celery/celery#8354] [Ideas]
Assume [5.3.0's release](https://docs.celeryq.dev/en/v5.3.1/changelog.html#version-5-3-0), the final release notes could be improved to follow a structure (perhaps like [Django's](https://docs.djangoproject.com/en/4.2/releases/4.2/)?)

- Curate by importance, in sections: e.g. breaking change, bug fix, new feature / setting, package updates
- For tagged branches, e.g. `5.3.1`, skip beta / release candidates for final releases unless its a `stable` branch
- Potentially also include a blog post of what the valuable changes are, and what is coming in the next release

**Comment by auvipy** [comment]
yes that would be good to follow. we will do that in next releases. https://docs.celeryq.dev/en/v5.3.1/whatsnew-5.3.html#whatsnew-5-3 is some overview
